src/Manager/Plugins.php

```diff
@@ -113,14 +113,14 @@ public function getPluginsByType($type)
 
     public function getAvailablePlugins()
     {
-        return VoyagerFacade::getJson(File::get(realpath(__DIR__.'/../../plugins.json')), []);
+        return VoyagerFacade::getJson(File::get(dirname(__DIR__, 2) . '/plugins.json'), []);
```

Comment:
Any particular reason not to use realpath?

Reply:
Mainly just simplicity IMO & an artifact of the inspections provided by my IDE. When I was starting as a system administrator the trainer instilled an idea of the using "the smallest hammer", so I like to apply that everywhere.

To me, the way `realpath` works and additional transformations it does is more more complex than `dirname`.

So if we were dealing with a path that included multiple relative folder changes that's when `realpath` would feel more necessary to me. So for example a constructed file/folder path which may end up looking quite nasty, like: `'/home/site-user/public_html/./some-content/assets/js/../css/'`.

Yet when the objective is simply ascending the current directory then `dirname` is the "smallest hammer" for my taste.